docs/Custom_Command_Keybindings.md

```diff
@@ -49,6 +49,12 @@ customCommands:
         filter: '.*{{index .PromptResponses 0}}/(?P<branch>.*)'
         valueFormat: '{{ .branch }}'
         labelFormat: '{{ .branch | green }}'
+  - key: '<f1>'
+    command: 'git reset --soft {{.CheckedOutBranch.UpstreamRemote}}'
+    context: 'files'
+    prompts:
+      - type: 'confirm'
+        initialValue: "Are you sure you want to reset HEAD to {{.CheckedOutBranch.UpstreamRemote}}?"
```

Comment:
the name `initialValue` suggests that it's something the user can modify but `body` would be more appropriate. What do you reckon?

Reply:
I debated the name when making the original changes, as well. My original thinking was that re-using `initialValue` might be better the perspective of flattening the list of available names for the user to understand, but I do agree it's confusing.

I refactored the code to add a new `body` field and think that it reads a bit better. Let me know what you think and I can either keep or revert the commit: 4f5e565b916b7c59484f5a7c45857d5179b49036